less/variables.less

```diff
@@ -13,14 +16,16 @@
 @footer-font-color: @waterloo;
 
 // Color Vars
-@wild-sand: #F6F6F6;
-@river-bend: #454A59;
-@sea-shell: #F1F1F1;
-@ebony-clay: #23293B;
-@shuttle-gray: #5E6164;
-@porcelain: #F6F7F8;
-@waterloo: #81848E;
-@btn-bg-color: #22283A;
+@wild-sand: #f6f6f6;
+@river-bend: #454a59;
+@sea-shell: #f1f1f1;
+@ebony-clay: #23293b;
+@shuttle-gray: #5e6164;
+@porcelain: #f6f7f8;
+@waterloo: #81848e;
+@btn-bg-color: #22283a;
 @border-real: #dee2e6;
 
 // Mobile var
+@media: (max-width 500px);
```

Comment:
The  @media keyword is reserved so this top line won't act as an escaped string, but the bottom one is